consensus/istanbul/backend/announce.go

```diff
@@ -889,7 +935,9 @@ func (sb *Backend) sendVersionCertificateTable(peer consensus.Peer) error {
 		logger.Warn("Error encoding version certificate msg", "err", err)
 		return err
 	}
-	return peer.Send(istanbul.VersionCertificatesMsg, payload)
+
+	sb.Unicast(peer, payload, istanbul.VersionCertificatesMsg)
```

Comment:
this means, this can fail, but we won't notice and we won't do anything about it... now, is that we what we want?

Additionally, we won't block waitinf for the send. also not sure if that's what we want

Reply:
Ya, good point.  It shouldn't be done asynchronously.